homeassistant/components/fortios/device_tracker.py

```diff
@@ -68,8 +68,8 @@ class FortiOSDeviceScanner(DeviceScanner):
 
     def __init__(self, fgt, fos_major_version, api_url) -> None:
         """Initialize the scanner."""
-        self._clients = {}
-        self._clients_json = {}
+        self._clients: dict = {}
+        self._clients_json: dict = {}
```

Comment:
Please make the dict complete e.g. dict[str, str]

Reply:
### self._clients
`self._clients` is a dict or a list. It's never used as a dict so I can't make it complete. I have changed it to `dict | list`.

### self._clients_json
`self._clients_json`is a dict but can also be an int. I can't make it complete because I don't have the `self._fgt.monitor(self._api_url, "")` return type.